modules/claims_api/config/schemas/v2/526.json

```diff
@@ -159,7 +159,7 @@
           "type": "object",
           "properties": {
             "beginDate": {
-              "description": "Date in MM-DD-YYYY the Veteran changed address. Date must be in the future if change is temporary.",
+              "description": "Date in MM-DD-YYYY the Veteran changed address.",
```

Comment:
new descriptions from Kayla:
Begin date for the Veteran's new address.

Reply:
are these resolved? i see outdated. @stiehlrod @acovrig